.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "advanced/digital-car-3d"]
```

Comment:
I'm not sure if it's a good idea to add the digital car as a submodule in the docs... It's quite heavy on the LFS data size, could potentially cause a lot of download traffic. Maybe it's enough to just add a link and let user download it themselves?

Reply:
I totally agree with you! Let me refactor this.

> Hi @hassan-pdx ! I really like the article! The trace player is awesome :) I did some cosmetic improvement suggestions, but left them as a separate PR into your branch so that you can review and discard/adapt what you like: #38

Hi @violinyanev, I'm glad you like it : )
Your improvements are much appreciated, I've just added a little formatting as well, and going to merge yours into this.